x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/rule_tab.tsx

```diff
@@ -14,39 +14,44 @@ import { RulesDetectionRuleCounter } from '../../rules/rules_detection_rule_coun
 import { CisKubernetesIcons, CspFlyoutMarkdown } from './findings_flyout';
 
 export const getRuleList = (
-  rule: CspFinding['rule'],
+  rule?: CspFinding['rule'],
   ruleState = 'unmuted',
   ruleFlyoutLink?: string
 ) => [
   {
     title: i18n.translate('xpack.csp.findings.findingsFlyout.ruleTab.nameTitle', {
       defaultMessage: 'Name',
     }),
-    description: ruleFlyoutLink ? (
-      <EuiToolTip
-        position="top"
-        content={i18n.translate('xpack.csp.findings.findingsFlyout.ruleTab.nameTooltip', {
-          defaultMessage: 'Manage Rule',
-        })}
-      >
-        <EuiLink href={ruleFlyoutLink}>{rule.name}</EuiLink>
-      </EuiToolTip>
-    ) : (
-      rule.name
-    ),
+    description:
+      ruleFlyoutLink && rule?.name ? (
```

Comment:
looks like exactly the same component from the overview_tab

Reply:
overall as we have two flyouts with similar info, it might benefit from sharing components between them. Otherwise I can see how we fix smth in one places and forget about another place